i18n/en.json

```diff
@@ -4,5 +4,10 @@
 			"Fannon"
 		]
 	},
-	"qrlite-desc": "Generates QRCodes in .png or .svg format on-the-fly."
+	"qrlite-desc": "Generates QRCodes in .png or .svg format on-the-fly.",
+	"qrcodes": "QR Codes",
+	"qrlite-specialpage-url-label": "URL:",
```

Comment:
It seems better to not have a colon in the label

Reply:
Good idea, done.